### PR TITLE
Deprecate some CRDS from the kcp_syncer

### PR DESCRIPTION
**Pull request by Roming22**
Signed-off-by: Romain Arnaud <rarnaud@redhat.com>

### CODE REVIEW

**Review by ramessesii2 — APPROVED** [review]
lgtm

File: images/kcp-registrar/register.sh [hunk 1]
```diff
@@ -130,7 +130,7 @@ register() {
             syncer_manifest="/tmp/syncer-${clusters[$i]}.yaml"
             KUBECONFIG=${kcp_kcfg} kubectl kcp workload sync "${clusters[$i]}" \
                 --syncer-image ghcr.io/kcp-dev/kcp/syncer:$KCP_SYNC_TAG \
-                --resources deployments.apps,services,ingresses.networking.k8s.io,conditions.tekton.dev,pipelines.tekton.dev,pipelineruns.tekton.dev,pipelineresources.tekton.dev,tasks.tekton.dev,runs.tekton.dev \
```
Comment (fgiloux) [inline images/kcp-registrar/register.sh:133]: deployments is needed when we want to deploy things like the gateway

File: ckcp/openshift_dev_setup.sh [hunk 2]
```diff
@@ -14,10 +14,10 @@ WORK_DIR="$SCRIPT_DIR/work"
 KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
 KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/admin.kubeconfig.base"
 CR_TO_SYNC=(
-            conditions.tekton.dev
+            services
```
Comment (bnallapeta) [inline ckcp/openshift_dev_setup.sh:17]: Since deployments are present in register.sh, we should include them here too to maintain consistency.

**Comment by Roming22** [comment]
Rebased, ready to be merged.

**Pull request merged**